src/components/features/Product/ProductDetailSection/index.tsx

```diff
@@ -0,0 +1,43 @@
+import { Box, Divider, Flex, Heading, Image, Text } from '@chakra-ui/react';
+
+import type { GoodsData } from '@/types';
+
+export const ProductDetailSection = (data: GoodsData) => {
```

Comment:
```suggestion
interface ProductDetailSectionProps {
  name: string;
  imageURL: string;
  price: {
    sellingPrice: number;
  }
}

export const ProductDetailSection = (props: ProductDetailSectionProps) => {
```

GoodsData에는 여러 속성이 있지만 컴포넌트 내부에서는 3가지 속성만 소비하고 있네요. 더 넓은 타입으로 인해 소비하지 않은 데이터에 대한 의존성을 가지게 되는거라 컴포넌트에서 필요한 데이터 의존성만 따로 타입을 선언해서 사용하는게 좋겠습니다.

[결합도 (Coupling)](https://en.wikipedia.org/wiki/Coupling_(computer_programming))의 종류 중에 Stamp coupling (data-structured coupling)에 해당한다고 볼 수 있겠습니다.

Reply:
필요한 데이터의 타입만 interface로 작성해서 사용하는 방법이 있었군요. 감사합니다!